packages/graphql/src/utils/test/builders/relation-field-builder.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { RelationField } from "../../../types";
+import { Builder } from "./builder";
+
+export class RelationFieldBuilder extends Builder<RelationField, RelationField> {
```

Comment:
My usual question - does it need to be a builder? Are we likely to ever add or change anything following construction?

Reply:
Consistency and flexibility.

* If we have a few builders already, I would keep using them for the rest of dummy data, just to make all work the same
* A builder allows us slightly more flexibility for adding/modifying dummy data or changing its structure (quite common in tests) and I found them really useful in the past. It is true that we may not need that in the moment, and we could go with functions, in this case, I would advocate for moving all of these to functions (again, consistency) rather than mixing different types of dummy data generators.

I'm happy to do a review on the builders that exists at the moment to decide whether to go for functions or builders (or other option) but I would really keep them consistent